src/relay/ir/alpha_equal.cc

```diff
@@ -361,6 +369,29 @@ class AlphaEqualHandler:
     }
   }
 
+  bool VisitExpr_(const RefCreateNode* op, const Expr& e2) final {
```

Comment:
nit: The meaning of the name `e2` is not that obvious in the first glance. Maybe rename `op` and `e2` to `lhs` and `other` to be consistent with the existing functions?

Reply:
Reference is merged independently. I will upload a PR for this.